hca/api/templates/api.jinja

```diff
@@ -0,0 +1,16 @@
+from ...added_command import AddedCommand
+
```

Comment:
add a comment that is put into the autogenerated files telling people not to edit the autogenerated f iles.